jme3-core/src/main/java/com/jme3/system/AppSettings.java

```diff
@@ -761,6 +764,16 @@ public void setResolution(int width, int height) {
         setHeight(height);
     }
 
+    /**
+     * Set the size of the window
+     * 
+     * @param width The width in pixels (default = resolution width)
+     * @param height The height in pixels (default = resolution height)
```

Comment:
It's not clear what "resolution width" and "resolution height" are. Perhaps you mean the frame-buffer width and height?